content/contributing/documentation.rst

```diff
@@ -15,407 +15,276 @@ This introductory guide will help you acquire the tools and knowledge you need t
 documentation, whether you plan to make a minor content change or document an application from
 scratch.
 
-.. _contributing/rst-intro:
+.. seealso::
+   :doc:`Discover other ways to contribute to Odoo <../contributing>`
+
+Read the :ref:`introduction to the reStructuredText language <contributing/documentation/rst-intro>`
+if you are not familiar with it. Then, you have two courses of action to start contributing to the
+documentation, depending on whether you want to propose minor changes to existing content or you
+instead want to work on significant changes to new and existing content.
+
+- **For minor changes**, for example, adding a paragraph or fixing a typo, we recommend that you
+  **use the GitHub interface**. This is the easiest and fastest way to submit your changes, and it
```

Comment:
```suggestion
- **For minor changes**, for example, adding a paragraph or fixing a typo, we recommend
  **using the GitHub interface**. This is the easiest and fastest way to submit your changes, and it
```

Reply:
Indeed. I choose this form to reference the "Use the GitHub interface" tab below but I realize that I renamed it at some point :sweat_smile: